### ISSUE TITLE
ModalInstance Close could be renamed to CloseAsync to following async naming conventions

### ISSUE THREAD

**Issue by ThaRaven403** [Blazored/Modal#206]
Hi Chris,

Noticed with the latest changes that the Close method of the ModalInstance is now an awaitable Task. I'd suggest renaming it to CloseAsync so that we don't need the intellisense to know we're dealing with an awaitable method, which would following naming conventions we often see for awaitable methods.

If you like the idea, I can even go ahead and open a PR for it !

**Comment by chrissainty** [comment]
Yes, agreed. Unfortunately, we've just pushed a new major version and I don't want to so that again so soon just for a name change. We'll look at adding this the next time we do a major release.

**Comment by ThaRaven403** [comment]
Good point. I'll still send you the PR and let you see when that can be merged.

On a side note, this new version works well!

**Comment by Kazbek** [comment]
Just wanna ask. In repo description you have example for [Multiple Modals](https://github.com/Blazored/Modal#multiple-modals) and we have this example:
```
async Task Yes()
    {
        var confirmationModal = ModalService.Show<ConfirmationPrompt>();
        var result = await confirmationModal.Result;

        if (result.Cancelled)
            return;

        BlazoredModal.Close(ModalResult.Ok($"The user said 'Yes'"));
    } 
```
This method marked as _async_ but _BlazoredModal.Close(ModalResult.Ok($"The user said 'Yes'"))_ calling without _await_. Is there a specific reason for this, or is it better to use _await_?

**Comment by chrissainty** [comment]
@Kazbek This is just an oversight and the call should be awaited.

**Comment by chrissainty** [comment]
This is now resolved by #258
